pages/api/deleteEngagement.js

```diff
@@ -0,0 +1,15 @@
+import { deleteEngagement } from "../../util/deleteEntry";
+
+export default async function handler(req, res) {
+    console.log(req?.body)
+    if (req.method !== 'POST') {
+      res.status(405).send({ message: 'Only POST requests allowed' })
+    }
+    try {
+        console.log(req.body);
+        const response = deleteEngagement(req.body);
```

Comment:
need to add **await** before call so that the page refreshes successfully (i.e. 
`const response = **await** deleteEngagement(req.body);`
). Should do it for all the other delete endpoints